docs/README.md

```diff
@@ -0,0 +1,41 @@
+# Code Reviews
+
+Code reviews are intended to help all of us grow as engineers and improve the quality of what we ship.
+These guidelines are meant to reinforce those two goals.
+
+## For reviewers
+
+Aim to respond to code reviews within 1 business day.
+
+Remember to highlight things that you like and appreciate while reading through the changes,
+and to make any other feedback clearly actionable by indicating if it is optional preference, an important consideration, or an error.
+
+Don't be afraid to comment with a question, or to ask for clarification, or provide a suggestion,
+whenever you don’t understand what is going on at first glance — or if you think an approach or decision can be improved.
```

Comment:
Suggestions from the reviewer on how to split a large PR into smaller chunks can also help move things along.

Reply:
will add that